tools/squashfskit4/Makefile

```diff
@@ -22,7 +22,7 @@ define Host/Compile
 	$(MAKE) -C $(HOST_BUILD_DIR)/squashfs-tools \
 		XZ_SUPPORT=1 \
 		LZMA_XZ_SUPPORT=1 \
-		XATTR_SUPPORT= \
+		XATTR_SUPPORT=1 \
```

Comment:
This doesn't look optional, did you check the size changes?

Reply:
This additional feature is inside a host tool, it doesn't go on the target. Knowing that, is a size increase really a big problem here ?

Reply:
No that shouldn't be a problem, thank you for the clarification.